src/smart-components/OpenChannelApp/Community.tsx

```diff
@@ -1,10 +1,11 @@
 import React, { ReactElement, useState } from 'react';
+import { OpenChannel } from '@sendbird/chat/openChannel';
 
 import './community.scss';
 import './theme.scss';
 
 import Sendbird from '../../lib/Sendbird';
-import OpenChannel from '../OpenChannel';
+import OpenChannelComponent from '../OpenChannel';
```

Comment:
nit: OpenChannelComponent  -> OpenChannelConversation  mgiht be agood naming

Reply:
changed!